sdk/turing/_base_types.py

```diff
@@ -0,0 +1,78 @@
+from datetime import datetime
+from turing.generated.model_utils import ModelNormal
+
+
+class ApiObject:
+    def __init__(
+            self,
+            id: int = 0,
+            created_at: datetime = None,
+            updated_at: datetime = None,
+            **kwargs):
+        self._id = id
+        self._created_at = created_at
+        self._updated_at = updated_at
+
+    @property
+    def id(self) -> int:
+        return self._id
+
+    @id.setter
+    def id(self, id: int):
```

Comment:
Why do we have these setters? Do we expect to use them or is it for completeness?

Reply:
For completeness